.github/workflows/post-merge.yml

```diff
@@ -0,0 +1,30 @@
+name: Post Merge
+
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
```

Comment:
As @jfharden suggested it would be super useful if you could add the concurrency setting here:

```
concurrency: nginx-proxy-post-merge
```

Reply:
Funny, for some reason my brain thought I'd already added this. Oh well, now it definitely is added!